va_explorer/users/models.py

```diff
@@ -84,3 +89,15 @@ def save(self, *args, **kwargs):
         # TODO: May need to be changed depending on how username comes in from ODK?
         self.username = self.email
         super(User, self).save(*args, **kwargs)
+        if self.original_password != self.password:
+            UserPasswordHistory.remember_password(self)
+
+
+class UserPasswordHistory(models.Model):
+    username = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
Could we call this field `user` to be consistent with other parts of Django?

Reply:
Yep, happy to make that change to match the convention. Implemented in 8e0ab3d